PathPlanning/DynamicWindowApproach/dynamic_window_approach.py

```diff
@@ -167,7 +167,8 @@ def calc_to_goal_cost(traj, goal, config):
     dx = goal[0] - traj[-1, 0]
     dy = goal[1] - traj[-1, 1]
     error_angle = math.atan2(dy, dx)
-    cost = abs(error_angle - traj[-1, 2])
+    cost_angle = error_angle - traj[-1, 2]
+    cost = abs(math.atan2(math.sin(cost_angle), math.cos(cost_angle)))
```

Comment:
What is the mathematical difference between previous one and current one?

Reply:
you wellcome.
for example (error_angle=-pi,   traj[-1, 2]=pi)  =====>>>>> cost = -2pi,
but is should be zero.